docs/ml-features.md

```diff
@@ -1098,9 +1098,9 @@ for more details on the API.
 
 `QuantileDiscretizer` takes a column with continuous features and outputs a column with binned
 categorical features. The number of bins is set by the `numBuckets` parameter.
-The bin ranges are chosen using an approximate algorithm (see the documentation for [approxQuantile](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala)
+The bin ranges are chosen using an approximate algorithm (see the documentation for [approxQuantile](api/scala/index.html#org.apache.spark.sql.DataFrameStatFunctions.scala)
 for a detailed description). The precision of the approximation can be controlled with the
-`relativeError` parameter. When set to zero, exact quantiles are calculated.
+`relativeError` parameter. When set to zero, exact quantiles are calculated. Computing exact quantiles is an expensive operation.
 The lower and upper bin bounds will be `-Infinity` and `+Infinity` covering all real values.
 
 **Examples**
```

Comment:
I believe the examples outlined below are no longer accurate with the change to using `approxQuantile` - unless we set `relativeError` to 0. Could we check that they still make sense? We could add something in this section saying, "if we compute exact quantiles, we should get the following DataFrame:"